addons/sys_core/fnc_findOcclusion.sqf

```diff
@@ -74,12 +74,11 @@ private _intersectObjects = lineIntersectsObjs [_startPos, _endPos, _vehicleUnit
 
 
 {
-    private _typeString = typeOf _x;
-    if (_typeString != "") then {
+    if (!isNull _x) then {
```

Comment:
I think we want to revert this change
it's for detecting map objects that don't have a type

Reply:
```suggestion
    if (typeOf _x != "") then {
```